### PR TITLE
danish up to date 14/8-16

### PR DESCRIPTION
**Pull request by boelle**

### CODE REVIEW

File: Marlin/language_da.h [hunk 1]
```diff
@@ -30,144 +30,238 @@
...
 #define LANGUAGE_DA_H
 
-#define MAPPER_C2C3
-// Define SIMULATE_ROMFONT to see what is seen on the character based display defined in Configuration.h
-//#define SIMULATE_ROMFONT
-#define DISPLAY_CHARSET_ISO10646_1
+//#define SIMULATE_ROMFONT //Comment in to see what is seen on the character based displays
+#if DISABLED(SIMULATE_ROMFONT) && DISABLED(DISPLAY_CHARSET_ISO10646_1) && DISABLED(DISPLAY_CHARSET_ISO10646_5) && DISABLED(DISPLAY_CHARSET_ISO10646_KANA) && DISABLED(DISPLAY_CHARSET_ISO10646_GREEK) && DISABLED(DISPLAY_CHARSET_ISO10646_CN)
+  #define DISPLAY_CHARSET_ISO10646_1 // use the better font on full graphic displays.
+#endif
```
Comment (Blue-Marlin) [inline Marlin/language_da.h:37]: The previous version was ok. The new one is 'en' only.

**Comment by boelle** [comment]
So i need to also remove the green lines?

File: Marlin/language_da.h (hunk 1 above)
Comment (thinkyhead) [inline Marlin/language_da.h]: @Blue-Marlin So… the original lines should be left as they were?

**Comment by thinkyhead** [comment]
The original lines should be left as they were. After all there is no reason in the Danish language file to look for Russian, Bulgarian, Kana, Chinese, Greek, etc., and certainly if `DISPLAY_CHARSET_ISO10646_1` is already enabled, then it's no harm to enable it again. These lines are only appropriate in the default (`language_en.h`) file.

**Comment by boelle** [comment]
I'm total lost here

What do i need to do where to make this one pass?

**Comment by zenmetsu** [comment]
I think the mapper/charset information in the prior version needs to be put back in place, judging from the conversation.  The file overall needs to be updated in the sense that messages are not all translated.  Any message string in English needs to be translated to Danish.  That is what needs to happen.

**Comment by boelle** [comment]
all strings are translated.... so i'm not any more wise than 3 days ago

**Comment by Blue-Marlin** [comment]
> So i need to also remove the green lines?

The opposite is the case. You have to let in the red ones and not to add the green ones..

**Comment by thinkyhead** [comment]
> What do i need to do where to make this one pass?

Apparently you need someone else to do it for you. I will fix this and post it for your edification.

Note that it's a bad idea to change your `RCBugFix` branch. Create a new branch before you make changes that you plan to submit as a PR.

```
git fetch upstream
git branch -D RCBugFix
git checkout upstream/RCBugFix -b RCBugFix
git checkout RCBugFix -b my_new_branch
```

…or just…

```
git fetch upstream
git checkout upstream/RCBugFix -b my_new_branch
```

**Comment by thinkyhead** [comment]
@boelle See #4649 for the patched-up version. It still can't be merged because some of the strings are far too long and must be reduced to 20 characters or fewer. Please provide some alternatives.

**Comment by zenmetsu** [comment]
@thinkyhead I saw a few that were over 20char.  Should they all be under 20char unless noted otherwise?  I updated language_ru.h and kept the ones I touched to 16char or less, but there was a prior one at 21char.

I saw your pull.  You stated all needed to be 20 or less.  I'll fix.

**Pull request closed without merging**